pkg/agent/multicast/mcast_discovery.go

```diff
@@ -90,20 +85,9 @@ func (s *IGMPSnooper) HandlePacketIn(pktIn *ofctrl.PacketIn) error {
 	return nil
 }
 
-func getInfoInReg(regMatch *ofctrl.MatchField, rng *openflow15.NXRange) (uint32, error) {
```

Comment:
If it is planned to be provided in package pkg/agent/openflow, maybe also update the function calls under path pkg/agent/controller/networkpolicy ?

Reply:
Agreed, but I think we could do this in another PR.

Reply:
I am OK to use a new patch for it.